applications/DG-Max/Programs/DGMaxEigen.cpp

```diff
@@ -254,8 +282,9 @@ class DGMaxEigenDriver : public AbstractEigenvalueSolverDriver<DIM> {
             // (or similarly predefined length). For the computation we assume
             // c=1, and assume the length scale a matches that of the mesh. Thus
             // the distance between x=0 and x=1 is assumed to be 'a'.
+            std::sprintf(freq_val, "%1.6f", frequency / (2 * M_PI) * lengthScale.getValue());
```

Comment:
I'm wondering why you switch here to the old c-style of printing. If you just want to set the precision then `<< std::setprecision(6)` would do the trick as well.